hyper/http20/connection.py

```diff
@@ -272,6 +273,8 @@ def request(self, method, url, body=None, headers=None):
         # being sent in the wrong order, which can lead to the out-of-order
         # messages with lower stream IDs being closed prematurely.
         with self._write_lock:
+            # TODO should we send an absolute url as in http11 ...
+            # ... when (self.proxy_host and not self.secure)?
```

Comment:
No, RFC 7540 doesn't need you to do it. The `:authority` header field is all that is needed.